sdks/wasm/DebuggerTestSuite/Support.cs

```diff
@@ -167,6 +169,7 @@ public Task Ready ()
 					dbgUrl = arrStr[0] + "/" + arrStr[1] + "/" + arrStr[2] + "/" + arrStr[arrStr.Length - 1];
 					dicScriptsIdToUrl[script_id] = dbgUrl;
 					dicFileToUrl[dbgUrl] = args["url"]?.Value<string>();
+					events.Add($"Debugger.scriptParsed {dicFileToUrl[dbgUrl]}");
```

Comment:
Do this for the `else` part too, to be consistently tracking `scriptParsed` events.